src/zmq/zmqnotificationinterface.cpp

```diff
@@ -72,7 +72,7 @@ CZMQNotificationInterface* CZMQNotificationInterface::CreateWithArguments(const
 // Called at startup to conditionally set up ZMQ socket(s)
 bool CZMQNotificationInterface::Initialize()
 {
-    LogPrint("zmq", "Initialize notification interface\n");
+    LogPrint("zmq", "zmq: Initialize notification interface\n");
```

Comment:
@laanwj maybe this should be done by `LogPrint`? Automatically prepend the category to the message?
Thoughts?